apps/web/components/apps/CalendarListContainer.tsx

```diff
@@ -109,12 +109,14 @@ export function CalendarListContainer(props: { heading?: boolean; fromOnboarding
                 {heading && (
                   <>
                     <DestinationCalendarSettingsWebWrapper />
-                    <SelectedCalendarsSettingsWebWrapper
-                      onChanged={onChanged}
-                      fromOnboarding={fromOnboarding}
-                      destinationCalendarId={data.destinationCalendar?.externalId}
-                      isPending={mutation.isPending}
-                    />
+                    <Suspense fallback={<SkeletonLoader />}>
```

Comment:
FYI i think this is happening because inside: 

packages/platform/atoms/selected-calendars/wrappers/SelectedCalendarsSettingsWebWrapper.tsx we are using in the calendar api query 

suspense:true

Reply:
That's the thing I didn't make it use suspense. It was there already
![image](https://github.com/user-attachments/assets/b34c23f0-b8f5-41bb-a3a6-a9564c541ea7)